lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -62,6 +62,7 @@ class QueryBuilder
      * @var array<string, mixed>
      */
     private $sqlParts = [
+        'distinct' => false,
```

Comment:
I'd move it one line below, it looks like there this array is ordered with the same order in which you would find these keywords in an SQL query.

Reply:
Looks like that. You are right